content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,186 @@
+*installation* or by adding a pod annotation.
+
+To simplify configuring network topology during installation create a single YAML file to pass to `istioctl`:
+
+{{< text yaml >}}
+cat <<'EOF' > topology.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      gatewayTopology:
+EOF
+{{< /text >}}
+
+{{< warning >}}
+If Istio ingress gateway was already running prior to application of the `MeshConfig` you will need
+to restart any Istio ingress gateway pods.
+{{< /warning >}}
+
+Both of the settings discussed can also be configured using the `proxy.istio.io/config` annotation to the Pod spec
+of your Istio ingress gateway.
+
+{{< text yaml >}}
+...
+  metadata:
+    annotations:
+      "proxy.istio.io/config": '{"gatewayTopology" : { "numTrustedProxies": 2 } }'
+{{< /text >}}
+
+### Configuring X-Forwarded-For Headers
+
+Many applications, both old and new, rely on client attributes, such as X-Forward-For header,
+to be forwarded by reverse proxies in a request. However, due to the variety of network
+topologies Istio can be deployed in, it is required to set the number of trusted proxies deployed in front
+of the Istio gateway proxy so that the client address can be extracted correctly.
+
+To set the number of trusted proxies add the following to your `topology.yaml` file.
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      gatewayTopology:
+        numTrustedProxies: <VALUE>
+{{< /text >}}
+
+For example, if you have a cloud based LB, a reverse proxy and then the Istio gateway proxy
+then `<VALUE>` would be 2.
+
+{{< warning >}}
+Note that all the proxies in front need of Istio gateway proxy must parse HTTP traffic and append the X-Forwarded-For
+headers at each hop. If the number of entries in X-Forwarded-For header is less than the number of
+trusted hops configured, Envoy falls back to using the immediate downstream address as the trusted
+client address. Please refer to [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for)
+to understand how X-Forwarded-For headers and trusted client addresses are determined.
+{{< /warning >}}
+
+See [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for)
+for more information about this capability.
+
+#### Example using XFF capability with httpbin
+
+1. Specify `numTrustedProxies` as 2 either through `MeshConfig` or an `proxy.istop/io/config` annotation. If using `MeshConfig`:
+
+    {{< text bash >}}
+    $ cat <<'EOF' > topology.yaml
+      apiVersion: install.istio.io/v1alpha1
+      kind: IstioOperator
+      spec:
+        meshConfig:
+          defaultConfig:
+            gatewayTopology:
+              numTrustedProxies: 2
+      EOF
+    $ istioctl manifest apply -f topology.yaml
+    {{< /text >}}
+
+1. Create `httpbin` namespace
```

Comment:
these steps need either a colon or period. I don't recall what the current practice is for our documentation. It used to be `:` but that may have changed. you can find out by reviewing other more recent documentation with steps that have gone through review.

Reply:
I've looked at a few other PRs, but there's still a mix of ':' and '.'. @frankbu?

Reply:
```suggestion
1. Create `httpbin` namespace with the following command:
```

Reply:
@jacob-delgado My suggestion follows the best practice.
Please let readers know in the text that the code-block is a command, and end each step with a colon (`:`).

Reply:
I did this for all steps. Thanks Rigs!